package lib.test.on2013_05.on2013_05_12_Codeforces_Round__183__Div__1_.A___Lucky_Permutation_Triple;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("src/lib/test/on2013_05/on2013_05_12_Codeforces_Round__183__Div__1_/A___Lucky_Permutation_Triple/A - Lucky Permutation Triple.task"))
			Assert.fail();
	}
}
